sync/sync_head.go

```diff
@@ -40,7 +40,7 @@ func (s *Syncer[H]) Head(ctx context.Context) (H, error) {
 		return s.Head(ctx)
 	}
 	defer s.getter.Unlock()
-	netHead, err := s.getter.Head(ctx)
+	netHead, err := s.getter.Head(ctx, header.WithDisabledSubjectiveInit(sbjHead))
```

Comment:
So basically you're passing subjective head to `WithDisabledSubjectiveInit`, right? :thinking:

Reply:
Yes, the naming is quite bad.